### ISSUE TITLE
chaos.jetzt serves user-excluding chat software 

### ISSUE THREAD

**Issue by KreerC** [chaos-jetzt/chaos-jetzt-ansible#20]
Currently, chaos.jetzt serves the screen reader-inaccessible riot-web Matrix client from its infrastructure. Even though Riot is developed under a Free Software license, it doesn't fully adhere to the four freedoms of Free Software and thus cannot be considered _it_. 

I suggest a replacement is found or an accessible alternative is promoted through group and website communication.

**Comment by DC7IA** [comment]
Do you have any suggestions?

**Comment by n0emis** [comment]
matrix-ircd (#17 ) could fix this problem. I am not really a fan of replacing matrix with something else, I would rather like to make it more accessible.

**Comment by KreerC** [comment]
The proposal was not to replace Matrix as such, but rather the Riot client. IRC bridging is sufficient, yes, but a technical burden not everyone can easily overcome.

**Comment by n0emis** [comment]
The problem here is, that there are not that many alternative clients, at least not for web. (Alternatives for desktop and mobile exist). 

I never really used IRC, although I know a relatively easy to use IRC-client: thelounge.chat 
Have you tried it and know, how accessible it is?

**Comment by YtvwlD** [comment]
Quassel is also pretty nice as an IRC client (at least the quassel-client, setting up a quassel-server may be a bit of a hassle).

**Comment by ruru4143** [comment]
I'm not a fan of mirroring every matrix room to an irc channel, because we have a [matrix-appservice-irc](https://github.com/matrix-org/matrix-appservice-irc) bridge not because we want to use irc, but because of old people
irc is depricated compared to matrix, it has a lack of a lot of very useful features like citing, fine utf-8 support or that fact that you don't need a bouncer

**Comment by e1mo** [comment]
Matrix can have nice and pretty features as one may like, if people are unable to use them those are just not available. And when IRC works for people as mentioned by @nkreer it's unfair to dismiss it as "old-mans chat service". We rather should, in my opinion, talk about how we can mirror new rooms in a efficient fashion.

**Comment by ruru4143** [comment]
I didn't mean to imply that we should exclude people. What i meant to say is that imho we should focus on a screen reader friendly web Matrix client instead of support a hole new communication network, which had its best time in 2002. 
If we can provide a screen reader compatible matrix web client, than we give people not only access to our matrix rooms, but access to the whole matrix communication network.
@nkreer Do you know if FluffyChat or tangent has better screen reader compatibility?
